lib: location: A-GPS improvements (#9703)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

Other libraries
---------------

* :ref:`lib_location` library:

  * Updated:

>     * GNSS filtered ephemerides are now not used when the :kconfig:option:`CONFIG_NRF_CLOUD_AGPS_FILTERED_RUNTIME` Kconfig option is enabled.

Reviewer: ```suggestion
    * GNSS filtered ephemerides are no longer used when the :kconfig:option:`CONFIG_NRF_CLOUD_AGPS_FILTERED_RUNTIME` Kconfig option is enabled.
```
